connection-limit/connection_limit_user.c

```diff
@@ -318,8 +274,8 @@ static int parse_tcpv6(int lnr, char *line)
 }
 
 /* Parse /proc/net/tcp and update bpf maps with needed data
-   map_fd[2]-> tcp_v4_conns: Holds TCP sockets in listen state
-   map_fd[0]->conn_count: Holds the number of concurrent inbound connections
+   tcp_conns_map_fd->tcp_v4_conns: Holds TCP sockets in listen state
```

Comment:
```suggestion
   tcp_conns_map_fd->tcp_v4_conns: Holds TCP4 sockets in listen state
```
consistency with line 193